data/src/main/java/com/microsoft/azure/kusto/data/UriUtils.java

```diff
@@ -44,4 +44,18 @@
 
         return false;
     }
+
+    public static String removeExtension(String filename) {
```

Comment:
is it really something you have to do manually? Do we not have it as part of the utils we already have? Especially since this only works for windows

Reply:
You could suggest something else
Anyway ill change to File.pathSeparator - so it will work on linux

Reply:
Ok, we'll avoid the extra dependency for now